soh/soh/Enhancements/randomizer/3drando/logic.cpp

```diff
@@ -582,6 +583,7 @@ namespace Logic {
     CanOpenStormGrotto  = CanPlay(SongOfStorms) && (ShardOfAgony || LogicGrottosWithoutAgony);
     HookshotOrBoomerang = CanUse(HOOKSHOT) || CanUse(BOOMERANG);
     CanGetNightTimeGS   = (CanPlay(SunsSong) || !NightGSExpectSuns);
+    CanBreakBeehives    = CanBreakBeehives || HookshotOrBoomerang || CanUse(BOW);
```

Comment:
I believe we may want to keep the flag out of the assignment, as the other helper methods are enough to satisify the check: 
```suggestion
    CanBreakBeehives    = HookshotOrBoomerang || CanUse(BOW);
```